web/portal/src/main/webapp/WEB-INF/web.xml

```diff
@@ -205,6 +196,11 @@
   </filter-mapping>
 
   <filter-mapping>
+    <filter-name>LocalizationFilter</filter-name>
+    <url-pattern>/*</url-pattern>
```

Comment:
This changes can have an impact on performances since the filter will now be triggered on all requests (even js, css, img, ...). Cannot it be more restrictive ?